pyaedt/edb_core/components.py

```diff
@@ -713,7 +713,7 @@ def create_source_on_component(self, sources=None):
         return True
 
     @pyaedt_function_handler()
-    def create_port_on_pins(self, refdes, pins, reference_pins, impedance=50.0):
+    def create_port_on_pins(self, refdes, pins, reference_pins, impedance=50.0, port_name=None):
         """Create circuit port between pins and reference ones.
 
         Parameters
```

Comment:
This entire "Parameters" section doesn't follow the standard format. It should be fixed.